lib/order.rb

```diff
@@ -0,0 +1,21 @@
+class Order
+  attr_reader :basket, :check, :all
+
+  def initialize
+    @basket = Hash.new(0)
+  end
+
+  def add_basket(item, num)
+    basket[item] += num
+  end
+
+  def basket_summary(menu)
+    @all = menu.dishes[item]*num
+    basket.map { |item,num| "#{item} x#{num} = £#{@all}" }.join(', ')
+  end
+
+  def bill(menu)
+    @check = basket.map { |item, num| @all }.inject(:+)
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `item`. You can omit all the arguments if you don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)<br>Lint/UnusedBlockArgument: Unused block argument - `num`. You can omit all the arguments if you don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)